src/utils/bytesToName.ts

```diff
@@ -0,0 +1,17 @@
+import { Interface, ParamType } from '@ethersproject/abi';
+import { BytesLike } from 'ethers';
+
+// import ParamType type
+
+function bytesToName(funcBytes: BytesLike, contractABI: string): string {
+  const iface = new Interface(contractABI!);
+  let name: string = '';
+  for (const key in iface.functions) {
+    if (funcBytes === iface.getSighash(key)) {
+      name = iface.functions[key].name;
```

Comment:
just need `return name;` here to early return once we found what we were looking for

Reply:
@hepnerthomas think u forgot this

Reply:
done